src/component/association-select.js

```diff
@@ -176,14 +179,17 @@ export class AssociationSelect {
     return this;
   }
 
+  resourceChanged() {
```

Comment:
It only does when the property is set.
```js
@bindable resource = null;
```
This will trigger the resource at initialization. Otherwise it won't.